core/field_image.js

```diff
@@ -47,6 +47,9 @@ goog.require('Blockly.utils.Size');
  */
 Blockly.FieldImage = function(src, width, height,
     opt_alt, opt_onClick, opt_flipRtl) {
+  Blockly.FieldImage.superClass_.constructor.call(
+      this, src || '', null);
```

Comment:
I'm not sure how this should be handled, but it feels weird to pass `src || ''` when `src` is required.

I don't have a better solution though :/

Reply:
src is a required field, but nothing about the constructor description says you can't pass null. ie: there's a difference between 
```
 * @param {string} src The URL of the image. Defaults to an empty string. 
``` and
```
 * @param {!string} src The URL of the image. Defaults to an empty string.
```

The former means you can still pass it null, hence the fallback to the empty string.